src/locales/en.ts

```diff
@@ -0,0 +1,836 @@
+    therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin:
+      'Saint Thérèse of the Child Jesus and the Holy Face of Lisieux, Virgin and Doctor of the Church',
+    therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin_copatroness_of_france:
+      'Saint Thérèse of the Child Jesus and the Holy Face of Lisieux, Virgin, Doctor of the Church and Copatroness of France',
+    thomas_apostle: 'Saint Thomas, Apostle',
+    thomas_aquinas_priest: 'Saint Thomas Aquinas, Priest and Doctor of the Church',
+    thomas_becket_bishop: 'Saint Thomas Becket, Bishop and Martyr',
+    thomas_hioji_rokuzayemon_nishi_priest_and_companions_martyrs:
+      'Saint Thomas Hioji Rokuzayemon Nishi, Priest, and Companions, Martyrs',
+    thomas_of_villanova_bishop: 'Saint Thomas of Villanova, Bishop',
+    thorfinn_of_hamar_bishop: 'Saint Thorfinn, Bishop',
+    thorlac_of_iceland_bishop: 'Saint Thorlac, Bishop',
+    timothy_of_ephesus_and_titus_of_crete_bishops: 'Saints Timothy and Titus, Bishops',
+    translation_of_the_relics_of_saint_stephen_of_hungary: 'Translation of the Relics of Saint Stephen of Hungary',
+    turibius_of_mogrovejo_bishop: 'Saint Turibius of Mogrovejo, Bishop',
+    ulrich_of_augsburg_bishop: 'Saint Ulrich of Augsburg, Bishop',
+    ursula_of_cologne_and_companions_virgins: 'Saint Ursula and Companions, Virgins and Martyrs',
+    valentine_of_raetia_bishop: 'Saint Valentine of Raetia, Bishop',
+    vincent_de_paul_priest: 'Saint Vincent de Paul, Priest',
+    vincent_ferrer_priest: 'Saint Vincent Ferrer, Priest',
+    vincent_kadlubek_bishop: 'Blessed Vincent Kadłubek, Bishop',
+    vincent_lewoniuk_and_companions_martyrs: 'Blessed Vincent Lewoniuk and Companions, Martyrs',
+    vincent_of_saragossa_deacon: 'Saint Vincent, Deacon and Martyr',
+    vincent_pallotti_priest: 'Saint Vincent Pallotti, Priest',
+    virgilius_of_salzburg_bishop: 'Saint Fergal, Bishop and Missionary',
+    visitation_of_mary: 'Visitation of the Blessed Virgin Mary',
+    vitus_of_lucania_martyr: 'Saint Vitus, Martyr',
+    vladimir_ghika_priest: 'Blessed Vladimir Ghika, Priest and Martyr',
+    vladimir_i_the_great_of_kiev: 'Saint Vladimir the Great',
+    waitangi_day: 'Waitangi Day',
+    walpurga_of_heidenheim_abbess: 'Saint Walpurga of Heidenheim, Abbess',
+    wenceslaus_i_of_bohemia_martyr: 'Saint Wenceslaus, Martyr',
+    wendelin_of_trier_abbot: 'Saint Wendelin, Abbot',
+    wilfrid_of_york_bishop: 'Saint Wilfrid, Bishop',
+    william_apor_bishop: 'Blessed William Apor, Bishop and Martyr',
+    willibald_of_eichstatt_bishop: 'Saint Willibald, Bishop',
+    willibrord_of_utrecht_bishop: 'Saint Willibrord, Bishop',
+    winefride_of_flintshire_virgin: 'Saint Winefride, Virgin',
+    wladyslaw_bladzinski_priest_and_companions_martyrs: 'Blessed Wladyslaw Błądziński, Priest, and Companions, Martyrs',
+    wladyslaw_of_gielniow_priest: 'Blessed Wladyslaw of Gielniow, Priest',
```

Comment:
@emagnier, we agreed that we won’t change the key names once we release v2.0.0. I have just found that `wladyslaw_of_gielniow_priest` should be renamed to `ladislas_gielniow_priest` and the key value to `*Ladislas*`. Besides `en.ts` (key name and its value), it should be changed in all locales + `poland.ts` (only key name).

I presume that you’d like to make this change in a separate PR, however, I think this change is quite small and the changes like this were already done in #213. Note that I am open for either option.